main.pm

```diff
@@ -857,6 +857,14 @@ else {
     }
     elsif (get_var("BOOT_HDD_IMAGE")) {
         loadtest "boot/boot_to_desktop.pm";
+        if (get_var("ISCSI_SERVER")) {
+            set_var('INSTALLONLY', 1);
```

Comment:
I just plant my suggestion into the brain of the next one to touch main.pm and it will work out - if not, I might be the one after that one ;)